### DISCUSSION TITLE
Disable default transaction in Postgres

### DISCUSSION THREAD

**Discussion by tiago-peixoto** [prisma/prisma#11603] [Q&A] [answered]
Is it possible to disable those transactions on every prisma query?

<img width="1072" alt="image" src="https://user-images.githubusercontent.com/1987829/152421225-27f8ed23-58bb-41a0-ace2-754ddb278fae.png">

**Comment by pantharshit00** [comment, marked as answer]
Hello, 

Currently, it is not possible to configure that behaviour. I suggest you to open a feature request about this here: https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=

**Reply by tiago-peixoto** [reply]
Thanks! Just opened it: https://github.com/prisma/prisma/issues/11634

I hope it makes sense.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!

**Reply by jonluca** [reply]
This should be reopened
